packages/api/src/utils.ts

```diff
@@ -75,3 +76,28 @@ export function groupBufficornsByRanch(
   }
   return bufficornsByRanch
 }
+
+export function generateUsernameList(count: number): Array<string> {
+  const usernames = new Set<string>()
+  // The seed must start at 1 because 0 means "use Math.random"
+  let counter = 1
+
+  while (usernames.size < count) {
+    let username = uniqueNamesGenerator({
+      dictionaries: [adjectives, names],
+      seed: counter,
+      separator: '-',
+      style: 'lowerCase',
+    })
+    usernames.add(username)
+    counter += 1
+  }
+
+  // Convert set into array to allow indexing by index
+  const usernameList = [] as Array<string>
+  usernames.forEach((value) => {
+    usernameList.push(value)
+  })
```

Comment:
You can use:
 - `spread operator`: const usernameList = [...usernames]
 -  `Array.from()`: const usernameList = Array.from(usernames)